Assets/Scripts/ManagerScripts/CountdownManager.cs

```diff
@@ -19,10 +19,15 @@ private void Awake()
         public AudioSource countdownSound;
         private bool _shouldCountTime;
         private bool _soundPlayed;
+        public GameObject playerGameObject;
+        private PlayerMovement _playerMovement;
+        private bool _hasSetPlayerInputToTrue;
 
         private void Start()
         {
+            _playerMovement = playerGameObject.GetComponent<PlayerMovement>();
             _shouldCountTime = SceneManager.GetActiveScene().name != "MainMenuScene";
+            _hasSetPlayerInputToTrue = false;
```

Comment:
lmao we're gonna have 5 state variables in this class ;_;

Reply:
changed this c: